modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase

```diff
@@ -117,10 +76,10 @@ definition {
 		}
 	}
 
-	@description = "LPS-164733 - Verify that Only the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
+	@description = "LPS-164733 - Verify that the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
```

Comment:
Fix to "LPS-164733 - Verify that the Finance user can not see 'Review button' when organization is in 'Awaiting Approval On EVP' status"

Reply:
I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

Reply:
I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985